operator/utils/url_expiration_duration.go

```diff
@@ -0,0 +1,42 @@
+package utils
+
+import (
+	"errors"
+	"os"
+	"time"
+)
+
+type ControllerType int
+
+const (
+	ScanController ControllerType = iota
+	HookController
+	ParserController
+)
+
+func (e ControllerType) String() string {
+	switch e {
+	case ScanController:
+		return "SCAN"
+	case HookController:
+		return "HOOK"
+	case ParserController:
+		return "PARSER"
+	default:
+		return "WRONG_ENUM_NUMBER"
+	}
+}
+
+func GetUrlExpirationDuration(controller ControllerType) (time.Duration, error) {
+	urlExpirationTimeString, envOk := os.LookupEnv("URL_EXPIRATION_" + controller.String())
+	if !envOk {
+		// env varible not set, use an hour as default
+		return time.Hour, nil
```

Comment:
I think one hour as the default is too low. I would stick with the current default of 12 hours unless there is a reason to change it.

Reply:
Ah sorry didn't realize that there were other specifc defaults set in the helm values. I guess then this is ok.
We should mention it though as changing the default duration for scans from 12 to 24 hours is a slight change to the security model